i18n.ts

```diff
@@ -2,6 +2,7 @@ import { getFixedT, init } from "i18next";
 import en from "./locales/en.json" with { type: "json" };
 import ja from "./locales/ja.json" with { type: "json" };
 import ko from "./locales/ko.json" with { type: "json" };
+import zh-CN from "./locales/zh-CN.json" with { type: "json" };
```

Comment:
The variable name `zh-CN` includes a hyphen, which is not allowed in JavaScript variable names. To resolve this, the variable name should be changed to `zhCN` or any other valid identifier:

```suggestion
import zhCN from "./locales/zh-CN.json" with { type: "json" };
```